src/main/java/frc/robot/Robot.java

```diff
@@ -166,36 +166,27 @@ private void configureBindings() {
     hardware
         .driverController
         .rightBumper()
-        .onTrue(robotCommands.passCommand())
+        .onTrue(robotCommands.feedingCommand())
         .onFalse(robotCommands.stopShootingCommand());
     hardware
         .driverController
         .leftBumper()
-        .onTrue(robotCommands.feedingCommand())
-        .onFalse(robotCommands.stopShootingCommand());
-
-    hardware
-        .operatorController
-        .rightTrigger()
-        .onTrue(robotCommands.ampCommand())
+        .onTrue(robotCommands.outtakeCommand())
         .onFalse(robotCommands.stopShootingCommand());
+    // TODO: Podium command?
```

Comment:
podium preset should be implemented the same as subwoofer, just different name/arm angle/shooter rpm

we already have some of it written (i think shooter has a state already?), just need to finish adding it